content/blog/getting-started-with-hpe-greenlake-api-for-data-services.md

```diff
@@ -0,0 +1,221 @@
+
+![](/img/async-operations-invocation-parameters.png "Execution of async-services using filter, sort, and select parameters")
+
+T﻿he response from the execution of GET /data-services/v1beta1/async-operations is provided below.
+
+![](/img/response-from-the-tryit-test.png)
+
+The corresponding task message from the GreenLake task’s UI is shown below.
+
+![](/img/corresponding-task-for-the-async-operations.png)
+
+### active-issues
+
+This API provides the list of the issues that require attention by the GreenLake users. The GreenLake UI provides a bell icon on the top right of every window (please see the previous paragraph under “What are these data services resources?” in this blog) to access the issues from every service available in the GreenLake. To limit the display of the properties’ response returned by this API, user can use parameter **select** as part of the header of the API execution. However, there are minimal set of properties required by **active-issues** API to be entered into the **select** parameter as shown in below response from active-issues API.
+
+```json
+https://<region-baseUrl>/data-services/v1beta1/issues?select=body
+{
+    "error": "Missing required field(s) - \"lastOccurredAt,customerId,createdAt,id,resourceUri,generation,type\"",
+    "errorCode": "422",
+    "traceId": "18e7b6adf90315de57f2b177652e3649"
+}
+```
+
+To alleviate this condition, the user can add the required properties to the **select** parameter in addition to any other property that is desired, as shown below.
+
+```json
+https://<region-baseUrl>/data-services/v1beta1/issues?select=body,lastOccuredAt,customerId,createdAt,id,resourceUri,generation,type
+```
+
+The completed execution of this API is shown below.
+
+![](/img/execution-result-from-active-issues.png)
+
+The above figure shows the output from GET issues given the parameter: “select body, createdAt, customerId, generation, id, lastOccuredAt, resourceURI, type, updatedAt”.  
+
+### Completing POST method for REST API using async-operations API with task id
+
+Any GreenLake REST API for data services with **POST** or **PATCH** methods (e.g. /virtualization/v1beta1/virtual-machines) will be executed asynchronously. The asynchronous execution means that execution of the API will complete and return with response = 0x202 status. The POST REST API process will run in the background; nonetheless, this operation must be monitored until it comes to completion. To accomplish that monitoring, the user will receive a task id in the header of the response, under location field, that had been successfully executed. The user will poll that task id using the GET {{baseUrl}}/data-services/v1beta1/async-operations/{Task Id} to retrieve the progress and status of the completion. Below is an example of this use case, where I executed the creation of virtual machines at the on-premises hypervisor.
```

Comment:
Suggest to add some line spaces to make the reading easier for the readers.
For example add a line space before "To accomplish..."
And a line space before "Below is an example..."

You may want to use backtick to make the API call a command
`GET {{baseUrl}}/data-services/v1beta1/async-operations/{Task Id}`